docs/api_docs/classes/great_expectations-data_context-data_context-data_context-DataContext.md

```diff
@@ -0,0 +1,63 @@
+---
+title: class DataContext
+---
+[See it on GitHub](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/data_context/data_context/data_context.py)
+
+### Docstring
```

Comment:
we can remove this header

Reply:
Fixed.  Shuffled the docstring up into the synopsis section for classes and methods, as well.